config/configuration.go

```diff
@@ -199,6 +196,10 @@ func init() {
 	viper.AddConfigPath("$PIPELINE_CONFIG_DIR/")
 	viper.SetConfigName("config")
 
+	// Deprecated
```

Comment:
should include suggestion about what to use instead

Reply:
Done. See also #2201